TS Error after upgrading to the latest react-router

I've started getting this error after upgrading types and react-router to their latest version:

`node_modules/use-react-router-breadcrumbs/dist/index.d.ts:27:77 - error TS2312: An interface can only extend an object type or intersection of object types with statically known members.`

--- a/package.json
+++ b/package.json
@@ -55,7 +55,7 @@
     "pinst": "^3.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.4.2",
     "rollup": "^2.79.0",
     "rollup-plugin-size": "^0.2.2",
     "rollup-plugin-terser": "^7.0.2",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "use-react-router-breadcrumbs",
-  "version": "4.0.0",
+  "version": "4.0.1",
   "description": "A hook for displaying and setting breadcrumbs for react router",
   "main": "dist/cjs/index.js",
   "module": "dist/es/index.js",

--- a/src/index.tsx
+++ b/src/index.tsx
@@ -75,12 +75,11 @@ export interface BreadcrumbComponentProps<ParamKey extends string = string> {
 export type BreadcrumbComponentType<ParamKey extends string = string> =
   React.ComponentType<BreadcrumbComponentProps<ParamKey>>;
 
-export interface BreadcrumbsRoute<ParamKey extends string = string>
-  extends RouteObject {
+export type BreadcrumbsRoute<ParamKey extends string = string> = RouteObject & {
   children?: BreadcrumbsRoute[];
   breadcrumb?: BreadcrumbComponentType<ParamKey> | string | null;
   props?: { [x: string]: unknown };
-}
+};
 
 export interface BreadcrumbData<ParamKey extends string = string> {
   match: BreadcrumbMatch<ParamKey>;

